### PR TITLE
ReadME added

### PR DESCRIPTION
**Pull request by zemark-prog**

### CODE REVIEW

**Review by dirayser — COMMENTED** [review]
Я Dmytriiev)

File: README.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
 # zhyt-ios
 
+## Overview
+
+We are creating an electronic payment system that will work with our currency ("zhyt-coin") and will allow you to easily transfer money from one user to another. All you need to perform these operations is an iPhone.
+
+## Use case diagram
+
+![zhyt-coin](https://user-images.githubusercontent.com/54912616/163831250-e49168c8-bfee-4113-bc1f-ff559b081aa8.jpg)
```
Comment (nosovk) [inline README.md:9]: better use mermaid, it's currently supported in github

File: README.md [hunk 2]
```diff
@@ -1,3 +1,23 @@
...
+
+## Main clauses
+
+To get started, the user must register by entering their data. After that, his data is transferred to the administrator for verification, who can accept the user or reject the application. If the application to create an account is rejected, the user can make changes to their data and submit a request for a review of the case. After the user's approval, he can use the account.
+
+The administrator can also block the selected user at any time.
+
+The user can add some personal information, as well as change it. The main functionality of the user is to replenish their own balance with our zhyt-coin currency and transfer the selected amount to another user from a personal account.
+
+## Contributors
```
Comment (nosovk) [inline README.md:35]: add tech parameters
add userflows (kyc needed?)

File: README.md [hunk 3]
```diff
@@ -1,3 +1,39 @@
...
+
+The administrator can also block the selected user at any time.
+
+The user can add some personal information, as well as change it. The main functionality of the user is to replenish their own balance with our zhyt-coin currency and transfer the selected amount to another user from a personal account.
+
+## Tech stack
+**Backend:**
+* Node.js
+* MySQL
+* Express
```
Comment (nosovk) [inline README.md]: may we get rid of that sh**? fastify for example? https://www.fastify.io/

File: README.md [hunk 4]
```diff
@@ -1,3 +1,39 @@
...
+* Sequelize
```
Comment (nosovk) [inline README.md]: take a look at http://knexjs.org/ pls. It's much more flexible.

File: README.md [hunk 5]
```diff
@@ -1,3 +1,39 @@
...
+
+**iOS Front:**
```
Comment (nosovk) [inline README.md:29]: тут я вам не помогу. Я могу flutter поднять что-то с ним сделать, но с IOS у меня не ладится. Так что эта часть на ваше усмотрение

File: README.md [hunk 6]
```diff
@@ -1,3 +1,39 @@
...
+## Overview
+
+We are creating an electronic payment system that will work with our currency ("zhyt-coin") and will allow you to easily transfer money from one user to another. All you need to perform these operations is an iPhone.
+
+## Use case diagram
+
+![zhyt-coin](https://user-images.githubusercontent.com/54912616/163831250-e49168c8-bfee-4113-bc1f-ff559b081aa8.jpg)
+
+## userflow
+![userflow_zhyt](https://user-images.githubusercontent.com/54912616/164428262-8f1af5b4-ca03-4a6b-8821-223d9a916511.png)
```
Comment (nosovk) [inline README.md]: - how balance top up will work?
- editing profile does not require additional verification?
- what is user id in system? email?
- there is 2 fill registration boxes? mistaken?

Comment (zemark-prog) [reply README.md]: 1. The balance top up is a simulation of account replenishment from own card, so you just need to enter the amount and your account balance will increase.
2. I've corrected this, I agree that it's necessary to ask additional verification.
3. Yes, email is unique
4. Yes, forgot to delete, corrected.

**Comment by nosovk** [comment]
lets start coding!

**Pull request merged**